.github/workflows/test-linux.yml

```
@@ -35,7 +35,11 @@ permissions:
 jobs:
   test-linux:
     if: github.event.pull_request.draft == false
-    runs-on: ubuntu-24.04
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-24.04, ubuntu-24.04-arm]
```

nit: we could go back to `ubuntu-latest` now that it's an alias for `ubuntu-24.04` IIUC
```suggestion
        os: [ubuntu-latest, ubuntu-24.04-arm]
```

---

IMO it's better to be explicit and choose when we want to switch to 26.04. We had issues with automated updates in the past (new compiler may not work with the `main` branch).